File: Semester4/web/Practical Exam/PHP/sport/server/model/Team.php
<?php

class Team implements JsonSerializable{
    private int $id;
    private string $name;
    private string $homeCity;

    public function __construct(int $id, string $name, string $homeCity){
        $this->id = $id;
        $this->name = $name;
        $this->homeCity = $homeCity;
    }

    public function getId(): int{
        return $this->id;
    }

    public function getName(): string{
        return $this->name;
    }

    private function getHomeCity(): string{
        return $this->homeCity;
    }

    public function jsonSerialize(){
        return [
            "id" => $this->id,
            "name" => $this->name,
            "homeCity" => $this->homeCity,
        ];
    }

}
?>

File: Semester4/web/Practical Exam/JSP/sport/src/main/java/com/example/sport/model/Team.java
package com.example.sport.model;

public class Team {
    private int id;
    private String name;
    private String homeCity;

    public Team(int id, String name, String homeCity) {
        this.id = id;
        this.name = name;
        this.homeCity = homeCity;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getHomeCity() {
        return homeCity;
    }

    public void setHomeCity(String homeCity) {
        this.homeCity = homeCity;
    }
}
